scripts/pre-commit/verify_migration_scripts.py

```diff
@@ -0,0 +1,182 @@
+import subprocess
+import sys
+import re
+import logging
+from pathlib import Path
+import typing as tp
+from datetime import datetime
+
+logger = logging.getLogger("verify_migration_script")
```

Comment:
Does this short version works instead of setting up the stream handlers? 

```python
logging.basicConfig(level=logging.INFO, format="%(levelname)s - %(message)s") 
logger = logging.getLogger("...") 
```